docs/modules/ROOT/pages/how-to/manage-projects-and-namespaces.adoc

```diff
@@ -0,0 +1,62 @@
+= Manage Projects and Namespaces
+
+This page explains the various mechanisms available to create OpenShift Projects in APPUiO Cloud.
+
+In Red Hat OpenShift, projects and namespaces can be used interchangeably, except when listing them. Check the xref:how-to/list-namespaces.adoc[corresponding page] for more information.
+
+== Creating Projects
+
+Through the web console:: Click on the btn:[Create Project] button and fill the form to create a new project.
+
+Through the Command Line:: Use the `oc new-project` command.
+
+Through the API:: Use the `ProjectRequest` resource to create a new project.
+
+TIP: In all cases, the new Project is automatically assigned to the default organization of your user. See also <<Project Ownership>>.
+
+== Creating Namespaces
+
+Namespaces can be created in two ways:
+
+. With the command line `kubectl create namespace` (or `oc create namespace`)
+. Through the API, using the `Namespace` resource.
+
+NOTE: You can't create namespaces through the web console.
+
+Namespaces created with any of these mechanisms can be managed just like any other OpenShift projects.
+
+Just like projects, namespaces are assigned to a specific organization using the `appuio.io/organization` label. Namespaces without such label will be assigned to the default organization. The code snippet below shows the YAML required to add the label to a new namespace request:
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    appuio.io/organization: my-company # <1>
+  name: my-namespace
+----
+
+<1> The identifier of your organization must be specified here.
+
+CAUTION: If the user doesn't have a default organization set, creating a namespace without specifying an explicit organization will fail.
+
+== Project and Namespace Ownership
+
+OpenShift Projects and Namespaces are always assigned to an organization, specified by the `appuio.io/organization` label. Check the xref:explanation/organizations-and-rbac.adoc[Organizations and RBAC] page for more information.
+
+If your user doesn't have a default organization configured, you must specify the label when creating the namespace or project. If you don't know your default organization name, try this:
```

Comment:
> you must specify the label when creating the namespace or project

Is it even possible to create a Project when there is now default organization defined? I have in my mind that then only Namespaces with an explicit organization configured can be created? And creation of a project for another organization than the default organization afaik doesn't work, does it?

Reply:
No, it is not possible. I have clarified this.